### ISSUE TITLE
False-positive message "Mandatory expression expected" on short list syntax

### ISSUE THREAD

**Issue by cosminardeleanu** [phpmd/phpmd#727]
- PHPMD version: 2.8.1
- PHP Version: 7.2.2 and 7.1.27
- Installation type: composer
- Operating System / Distribution & Version: MacOs Mojave 10.14.6 and Debian 9

### Current Behavior
```
sample.php    -       The parser has reached an invalid state near line "2" in file "sample.php". Please check the following conditions: Mandatory expression expected.
```

```php
[, $variable2] = [1, 2];
```

### Expected Behavior
Should not trigger any errors. The code is perfectly valid.
Note: If we're using `list(, $variable2) = [1, 2];` then the code will throw no error.

### Steps To Reproduce:
Sample code from above.

**Comment by richardhughes** [comment]
Raised an issue with pdepend for this as it seems like its a upstream issue, see the linked issue :)

**Comment by kylekatarnls** [comment]
This should be fixed with pdepend/pdepend#476 and you can yet test it requiring "pdepend/pdepend": "dev-feature/issue-473-skipped-variable-in-unpacking"

**Comment by AntonEvers** [comment]
I think this can be closed now @kylekatarnls.
